docs/quick_start.md

```diff
@@ -78,20 +101,56 @@ Install [Clang](https://clang.llvm.org/get_started.html) or follow installation
 - Development headers of GTK 3 and libnotify:
 Follow installation instructions on the [electron docs](https://www.electronjs.org/docs/development/build-instructions-linux#prerequisites)
 
-** Note: Ubuntu 18.10 or later recommended (or another distro new enough to support GLIBC_2.28) **
+**Note: Ubuntu 18.10 or later recommended (or another distro new enough to support GLIBC_2.28)**
 
-### 4. Install Honeycomb npm packages
-Once [Node.js](https://nodejs.org/en/download/) is installed you will able to use `npm` commands in the terminal. To install the dependencies for Honeycomb run the following command at the terminal (remember you need `cd your-new-task-name` before)
+### 4. Add Your Personal Access Token
 
-```
-npm install
-```
+Honeycomb depends on certain packages only present in GitHub's own package manager. In order to install the packages you generate a personal access token on your GitHub account:
+
+- Create a new branch
+- Open the `.npmrc` file in `your-new-task-name` and add the following line:
+
+    ```
+    //npm.pkg.github.com/:_authToken=
+    ```
+
+- Log in to [GitHub](https://github.com) on the web
+- Navigate to Settings -> Developer Settings -> Personal Access Tokens
+- Click "Generate a new token", add a note, set `Expiration` to "No expiration" and select `read:packages`
+
+    ![Personal Access Token access settings](assets/github-pat-access.png)
+
+- Click "Generate Token" and copy the token. *The token will be lost forever after you exit the page!*
+
+    ![Personal Access Token access code](assets/github-pat-token.png)
+
+- Paste the code after the = from before and save. It should look as such:
+
+    ```
+    //npm.pkg.github.com/:_authToken=ghp_abcde12345
+    ```
+
+`.npmrc` now contains a secret key, we must tell git to stop tracking it.
+
+- Add ".npmrc" to the end of the `.gitignore` file
+- Remove the file from git:
```

Comment:
Maybe we should add `.npmrc` to our gitignore and no rely on users doing it.

Set the current `.npmrc` file in this repo to `.npmrc.example` and add the stub for this permission key.

Then tell the user to copy the file.
```
cp .npmrc.example .npmrc
```

This is something we do for the soma backend repo as well. Let me know what you think !

Reply:
Windows can be annoying with file extensions but if people are already in their IDE at this point I think that's a great idea!

Reply:
ooh good point

Reply:
I like this strategy. But if we are asking to add tokens to it, we should definitely have it in the `.gitignore`